### ISSUE TITLE
Killing renode when staticAnalyzer output match defined pattern

### ISSUE THREAD

**Issue by MSMazaya** [renode/renode#469]
Consider the following example of my `.resc` script.

```
using sysbus

mach create "K210"

machine LoadPlatformDescription @platforms/cpus/kendryte_k210.repl

path add @/

emulation CreateServerSocketTerminal 3456 'term'
connector Connect sysbus.uart term

showAnalyzer uart

sysbus Tag <0x50440000 0x10000> "SYSCTL"
sysbus Tag <0x50440018 0x4> "pll_lock" 0xFFFFFFFF
sysbus Tag <0x5044000C 0x4> "pll1"
sysbus Tag <0x50440008 0x4> "pll0"
sysbus Tag <0x50440020 0x4> "clk_sel0"
sysbus Tag <0x50440028 0x4> "clk_en_cent"
sysbus Tag <0x5044002c 0x4> "clk_en_peri"

# enable uart tx
uart WriteDoubleWord 0x8 0x1

macro reset
"""
    sysbus LoadELF $bin
    # Is there something like quitWhenAnalyzerMatch "END" that I can use to quit renode when the output of staticAnalyzer match something?
"""
runMacro $reset
```

I am using the above renode script for testing automation and reading its output via the socket terminal. Currently, I need to manually kill the process after getting the output from telnet. Is there any way I can kill the process automatically when an output of the analyzer met some defined pattern? Something like `Wait For Line On Uart` for `renode-test`, Thanks!

**Comment by MSMazaya** [comment]
Hi @WojMel

Thank you for the answer, I noticed that the hook does not work when I use `--disable-xwt`. Is there any additional things that I need to do for it to work with `--disable-xwt`?

**Comment by WojMel** [comment]
Hi @MSMazaya,

Quitting from hooks when using `--disable-xwt` should be fixed since 95d25ea. Could you verify it works for your use case?

**Comment by PiotrZierhoffer** [comment]
Closing. If the issue persists, please open a new one
